### PR TITLE
Fix/core schema minor

### PR DESCRIPTION
**Pull request by annikaLiving**
Added
* West Coast as an enum

Changed
* the state attribute name and values
* the status values

### CODE REVIEW

**Review by JTA2020 — COMMENTED** [review]
Curious why we are deleting  "Suspended" in applicaiton_lifecycle_state @annikaLiving

**Comment by annikaLiving** [comment]
@JTA2020 suspended was removed as On Hold existed and Suspended could mean On Hold in the process, or an issued permit could be suspended after approval for any number of reasons. feels ambiguous. more work with come on states and statuses as we compile our research. Then developing the data dictionary for all of these terms.

**Review by JTA2020 — APPROVED** [review]
LGTM, thanks for elaborating

**Pull request merged**
